modules/ROOT/pages/addition/index.adoc

```diff
@@ -150,8 +150,8 @@ For each managed DBMS instance on the host, the following environment variables
 | password
 |===
 
-To enable the log streaming functionality, there are also a few optional environment variables. 
-If `CONFIG_INSTANCE_n_QUERY_LOG_PORT` is not set, the feature is treated as disabled.
+To enable the query log collection functionality, there are also a few additional environment variables.
```

Comment:
Hmm yeah this might be for the best. What I originally wanted to convey with this wording was that there are several variables which are indeed _optional_ to set. Then again, the port is mandatory to enable the feature.